tests/test_main.py

```diff
@@ -179,38 +180,89 @@ async def test_authenticate_exceptions() -> None:
     assert str(excinfo.value) == "Timeout error: Request timed out"
 
 
-async def test_get_homes(
-    caplog: pytest.LogCaptureFixture, snapshot: SnapshotAssertion
+@pytest.mark.parametrize(
+    "fixture_file,method,expected_url,expected_logs",
+    (
+        (
+            "me.json",
+            "get_homeowner",
+            ME_URL,
+            (
+                '"email": "**redacted**"',
+                '"firstName": "**redacted**"',
+                '"lastName": "**redacted**"',
+                '"deviceSerialNumbers": [\n        "**redacted**"\n    ],\n',
+            ),
+        ),
+        (
+            "notifications.json",
+            "get_notifications",
+            NOTIFICATIONS_URL,
+            (
+                '"message": "The engine on your generator \\"mygenerator\\" has '
+                'stopped."',
+                '"serialNumber": "**redacted**"',
```

Comment:
Added this for consistency, now that there is a fixture for `/kem/api/v3/notifications`.